com.amphi.music.yaml

```
@@ -0,0 +1,67 @@
+id: com.amphi.music
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: music
+
+finish-args:
+  - --share=network
+  - --filesystem=home
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/bin/lib:$LD_LIBRARY_PATH
+  - --socket=x11
+  - --socket=wayland
+  - --talk-name=org.freedesktop.portal.Desktop
+  - --socket=pulseaudio 
+
+modules:
+
+  - name: music
+    buildsystem: simple
+    build-commands:
+      - mkdir -p external
+      - tar --use-compress-program=unzstd -xf music.tar.zst
+      - tar --use-compress-program=unzstd -xf external.tar.zst -C external
+      - install -Dm755 music /app/bin/music
+      - cp -r lib /app/bin/lib
+      - cp -r data /app/bin/data
+      - ls
+      - cp -r external/* /app/bin/lib/
+
+      - install -Dm644 com.amphi.music.desktop /app/share/applications/com.amphi.music.desktop
+      - install -Dm644 icon-128.png /app/share/icons/hicolor/128x128/apps/com.amphi.music.png
+      - install -Dm644 icon-256.png /app/share/icons/hicolor/256x256/apps/com.amphi.music.png
```

```suggestion
      - install -Dm644 icon-256.png /app/share/icons/hicolor/512x512/apps/com.amphi.music.png
```

the image is 512x512.